docs/maintainer-workflow.md

```diff
@@ -0,0 +1,70 @@
+# Maintainer guide
+
+Welcome to the maintainer guide for our project!
+As a project maintainer, you play a crucial role in ensuring the success and health of the project.
+Below are key responsibilities and guidelines to help you manage the project effectively.
+
+## Key responsibilities
+
+### Manage Issues
+1. Issue Triage
+    - Establish a regular schedule for reviewing and triaging open issues.
+    - Close irrelevant issues, and categorize and prioritize others using milestones.
+
+1. Acknowledging an Issue
+    - Review new issues promptly to address major blockers.
+    - Leave a comment on the issue to show appreciation for the creator's time and efforts.
```

Comment:
Does this make sense? - https://github.com/actions/first-interaction

Reply:
This action is just for first time contributors. We could add a link to contributor workflow doc and request them to take a look at it.

We could add this in #96, or create a separate issue for this.